src/Card/CardStatus.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Alert from '../Alert';
+
+const CardStatus = React.forwardRef(({
+  className,
+  children,
+  variant,
+  icon,
+}, ref) => (
+  <Alert
```

Comment:
We decided we will _not_ use the existing `Alert` for the following reasons:
* a11y requirements around screen readers (e.g., if there are many card with a Card.Status rendered at once).
* We would not like to create a dark variant for `Alert` to support this.

Instead, we'll replicate the `Alert` behavior without the a11y needs of it and all color variants of `Card.Status` as speced.

Reply:
Updated. Added css classes for `CardStatus` instead of using the `Alert` component